### PR TITLE
fix: replace obsolete icons

### LINKED ISSUE #4107: Some icons not showing (nvim-tree, bufferline, etc.)
### Problem description

Lots of icons are not showing properly.

### LunarVim version

release-1.3/neovim-0.9-e823ccb

### Neovim version (>= 0.8.0)

0.9.0

### Terminal name

Windows Terminal

### Operating system/version

Windows 11

### Steps to reproduce

Note: I use NerdFont v3.0.0, so that might be the issue. Because I don't see this issue being tracked on another issue, I might as well just open one.

1. Open lunarvim
2. Open two buffers
3. Open nvim-tree

### Screenshots

image

### PR DESCRIPTION
**Pull request by cpea2506**
# Description

As NerdFont recently released V3, there are breaking changes that cause some icons not showing. Those icons were replaced using [nerdfix](https://github.com/loichyan/nerdfix/tree/v0.2.2).

This can break anyone currently using V2.

Fixes #4107

## How Has This Been Tested?
...

### CODE REVIEW

**Comment by lunacd** [comment]
I can't really test whether this changeset is exhaustive. But all the changed ones look good with a NF 3.0 font.

**Comment by Hermanverschooten** [comment]
I just copied the file in-place on my mac with Meslo Nerd font and everything looks fine now. thx

**Comment by Hermanverschooten** [comment]
I saw there is some controversy about this change, as people believe v2 will still be around for a long time.
I was thinking, can this not be put with a config flag that defaults to v2, but can be se to use `nerd_fonts_version=3` or something?

**Comment by LostNeophyte** [comment]
> I saw there is some controversy about this change, as people believe v2 will still be around for a long time. I was thinking, can this not be put with a config flag that defaults to v2, but can be se to use `nerd_fonts_version=3` or something?

Impossible in config.lua, possible with an env variable

**Comment by Hermanverschooten** [comment]
@LostNeophyte Okay, I didn't know, I thought it would be possible to do something like `lvim.opt.nerd_font_version`.

**Comment by wsmbsbbz** [comment]
This causes my alpha.nvim screen cursor to point to the blank space in front of the text, I'm not sure why this is.
Screenshot 2023-05-04 22 29 16

**Review by lunacd — APPROVED** [review]

File: lua/lvim/icons.lua [hunk 1]
```diff
@@ -110,19 +110,19 @@ return {
...
     SignOut = "",
-    Tab = "",
+    Tab = "󰌒",
     Table = "",
     Target = "󰀘",
     Telescope = "",
     Text = "",
     Tree = "",
-    Triangle = "契",
+    Triangle = "󰐊 ",
```
[the remark this answers is not in the document]
Comment (cpea2506) [reply lua/lvim/icons.lua]: My terminal display the old `Triangle` with trailing space that make me think there is a space after. 😅

**Review by ChristianChiarulli — APPROVED** [review]

**Comment by ChristianChiarulli** [comment]
we also need this update in the 1.3 branch as well

**Comment by lunacd** [comment]
What would be the official process to backport this to 1.3 branch? Should I open a separate PR against 1.3 branch?

**Pull request merged**